xml/admin_ceph_upgrade.xml

```diff
@@ -931,83 +951,93 @@ mds standby for name = mds.456-another-mds
    </procedure>
   </sect2>
 
-<!--
-   <sect2 xml:id="upgrade-one-node-auto">
-    <title>Automated Node Upgrade using the <emphasis>SLE15 Migration</emphasis> Package</title>
-    <procedure>
-     <step>
-      <para>
-       Disable or remove all existing software repositories:
-      </para>
+  <sect2 xml:id="upgrade-one-node-auto">
+   <title>Node Upgrade using the &suse; Distribution Migration System</title>
+   <para>
+    The <emphasis>Distribution Migration System</emphasis> (DMS) provides an
+    upgrade path for an installed &sle; system from one major version to
+    another. The following procedure utilizes DMS to upgrade &productname;
+    &prevproductnumber; to version &productnumber; including the underlying
+    &prevcephos; to &cephos; migration.
+   </para>
+   <para>
+    Refer to
+    <link xlink:href="https://documentation.suse.com/suse-distribution-migration-system/1.0/single-html/distribution-migration-system/"/>
+    to find both general and detailed information about DMS.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Install the migration RPM packages. They adjust the &grub; boot loader to
+      automatically trigger the upgrade on next reboot. Install the
+      <package>SLES15-SES-Migration</package> and
+      <package>suse-migration-sle15-activation</package> packages:
+     </para>
+<screen>&prompt.sminion;zypper install SLES15-SES-Migration suse-migration-sle15-activation</screen>
+    </step>
+    <step>
+     <substeps>
+      <step>
+       <para>
+        If the node being upgraded <emphasis role="bold">is</emphasis>
+        registered with a repository staging system such as SCC, SMT, RMT, or
+        &susemgr;, create the
+        <filename>/etc/sle-migration-service.yml</filename> with the following
+        content:
+       </para>
 <screen>
-&prompt.root;zypper mr -d -a
+use_zypper_migration: true
+preserve:
+rules:
+- /etc/udev/rules.d/70-persistent-net.rules
```

Comment:
`rules:` and the subsequent line need one more indent as under "Preserve System Data" at https://documentation.suse.com/suse-distribution-migration-system/1.0/single-html/distribution-migration-system/#_optional_customization_of_the_upgrade_process, so this should be:

```
use_zypper_migration: true
preserve:
  rules:
    - /etc/udev/rules.d/70-persistent-net.rules
```

(Also, `use_zypper_migration: true` is the default setting, to technically doesn't need to be specified in this case, but there's no harm in leaving it there for symmetry with the other case)

Reply:
updated